pkgs/by-name/co/coconutbattery/package.nix

```diff
@@ -5,11 +5,15 @@
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "coconutbattery";
-  version = "3.9.14";
+  version = "3.9.16,278173DA";
```

Comment:
please split of the rev into a separate variable.

Reply:
resolved